pdf/config.json5

```diff
@@ -327,6 +355,10 @@
                         nameGroup: 'PDF Header',
                     },
 
+                    '^Date.*$': {
```

Comment:
What's the purpose of this one? Dates are handled a few groups down, in `'^[ \\t]*D:(\\d{14}).*(/[a-zA-Z0-9]+)$': {`

Also, I think it's worth separating the different date types. CreationDate and ModifiedDate are very distinct types of metadata.